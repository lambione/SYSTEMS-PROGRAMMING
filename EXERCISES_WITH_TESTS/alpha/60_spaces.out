QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQakQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQakQakQakQak
akQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQak
akQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQak
akQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
