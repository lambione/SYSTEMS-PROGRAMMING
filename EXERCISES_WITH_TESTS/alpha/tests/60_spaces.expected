QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQakQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQakQakQakQakQakQakQakQakQak
QakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQa
kQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQ
akQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQakQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
aQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQ
caQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQca
QcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQcaQc
a
